json_serializable/lib/src/type_helpers/enum_helper.dart

```diff
@@ -4,18 +4,22 @@
 
 import 'package:analyzer/dart/element/type.dart';
 
+import '../json_key_utils.dart';
 import '../json_literal_generator.dart';
 import '../type_helper.dart';
 import '../utils.dart';
 
 final simpleExpression = RegExp('^[a-zA-Z_]+\$');
 
-class EnumHelper extends TypeHelper {
+class EnumHelper extends TypeHelper<TypeHelperContextWithConfig> {
```

Comment:
It would be nice if changes like this were called out in the commit message - why was this added? How do the changes in this class contribute towards the goal?